test/metabase/test_setup.clj

```diff
@@ -113,11 +113,3 @@
               [symb varr] (ns-publics ns)
               :when       (not (:doc (meta varr)))]
           (symbol (str (ns-name ns) "/" symb)))))
-
-(defn- throw-if-metabase-doesnt-have-enough-docstrings!
-  {:expectations-options :before-run}
-  []
-  (when-let [things-that-need-dox (seq (things-that-need-dox))]
-    (println (u/format-color 'red "Every public var in Metabase might as well have a docstring! Go write some for the following (or make them ^:private):\n%s"
-               (u/pprint-to-str things-that-need-dox)))
-    (System/exit -1)))
```

Comment:
The check is in there to catch us when we're tempted to cut corners and write code that's not as high-quality as it ought to be. Our codebase is well-documented right now, so let's keep it that way. If not for our own sake, for the sake of all the people out there who would like to contribute to it.

At any rate I've moved this out into a separate linter.